android/core/src/main/java/com/coinbase/wallet/core/util/BoundedCache.kt

```diff
@@ -0,0 +1,36 @@
+package com.coinbase.wallet.core.util
+
+class BoundedCache<T, R>(private val maxSize: Int) {
```

Comment:
we don't need this in kotlin/java world. Use `ConcurrentLruCache`. I'm already using it in `WalletLinkWebSocket`

Reply:
Sweet